items/iptables.py

```diff
@@ -115,6 +116,7 @@ def generate_ignored_rules_and_chains(attributes):
             ignored_chains += [chain_name, ]
 
         for rule in attributes['chains'][chain_name].get('ignored_rules', []):
+            rule['chain'] = chain_name
```

Comment:
since this is changing the rule here, this should be done in a metadata retractor. and not magicaly in the bundle